sources/packages/web/src/views/aest/student/CASSupplierInformation.vue

```diff
@@ -18,6 +18,15 @@
                   prepend-icon="fa:fa fa-plus-circle"
                   >Edit info</v-btn
                 >
+                <v-btn
+                  class="mr-2 float-right"
+                  color="primary"
+                  data-cy="retryCASSupplierButton"
+                  :disabled="notAllowed || !retryButtonDisabled"
```

Comment:
Is the logic not the oposite?
```vue
:disabled="notAllowed || retryButtonDisabled"
```

Reply:
I believe if it is true it will disable the button, so i had the naming as retryButtonDisabled. Changed the logic accordingly now.